sdk/eventhub/Microsoft.Azure.WebJobs.Extensions.EventHubs/src/Config/EventHubOptions.cs

```diff
@@ -81,6 +82,12 @@ public int MaxBatchSize
         /// </summary>
         public bool InvokeProcessorAfterReceiveTimeout { get; set; }
 
+        /// <summary>
+        /// Gets the initial offset options to apply when processing. This only applies
+        /// when no checkpoint information is available.
+        /// </summary>
+        public InitialOffsetOptions InitialOffsetOptions { get; internal set; }
```

Comment:
I don't think we need internal setter.

Reply:
We use the setter in the tests.

Reply:
Ah I guess we can just set the properties directly.